content/en/docs/tasks/administer-cluster/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,47 @@
+
+### Unknown Version Interoperability Proxy Alphs
+
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+The Unknown Version Interoperability Proxy feature is controlled by a
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+and is not enabled by default. You can activate Unknown Version Interoperability Proxy by enabling
+the feature gate named `UnknownVersionInteroperabilityProxy` when you start the
+{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}:
+
+```shell
+kube-apiserver \
+--feature-gates=UnknownVersionInteroperabilityProxy=true \
+# required flag for this feature
+--peer-ca-file=`ca bundle to verify serving certs of kube-apiserver`
+# optional flags for this feature
+--peer-advertise-ip=`ip of this kube-apiserver that should be used by peers to proxy requests`
+--peer-advertise-port=`port of this kube-apiserver that should be used by peers to proxy requests`
+# …and other flags as usual
+```
+
+To support correct handling of resource requests that are made when a cluster has multiple kube-apiservers at varied versions, a new filter is added to the handler chain in the aggregator which proxies clients to an apiserver that is capable of handling their request.
+
+When a kube-apiserver receives a resource request, it first checks which apiservers can serve the requested resource using StorageVersion API. If the resource is known to the local apiserver, the request is handled locally.
+
+If not, the request is proxied to one of the identified kube-apiservers that are aware of the requested resource. If the identified peer apiserver fails to respond (due to reasons like network connectivity, race between the request being received and the controller registering the apiserver-resource info in ServerStorageVersion object), then a 503 is served.
+
+#### Proxy transport between apiservers and authn
+
+To authenticate the destination server's serving certs, a CA bundle passed to kube-apiserver's `--peer-ca-file` flag will be used. This is a new required flag for this feature to work since there is no default value enabled for this flag. 
+
+The source kube-apiserver will reuse the existing `--proxy-client-cert-file` and `--proxy-client-key-file` to present its identity which is verified by the peer (destination kube-apiserver) using the existing `--requestheader-client-ca-file` cert that is passed to kube-apiserver upon startup.
+
+#### Identifying destination apiserver's network location
+
+To find the network location of the peer kube-apiserver, the values passed to the newly introduced `--peer-advertise-ip` and `--peer-advertise-port` flags upon kube-apiserver startup are used. If these flags are unspecified, the value passed to either `--advertise-address` or `--bind-address` flag is used. If those too, are unset, the host's default interface will be used.
```

Comment:
```suggestion
To find the network location of the peer kube-apiserver, you use the 
 `--peer-advertise-ip` and `--peer-advertise-port` command line arguments to kube-apiserver.
If these flags are unspecified, the kube-apiserver uses the value from either `--advertise-address` or
`--bind-address` flag is used. If those too, are unset, the host's default interface will be used.
```
Can you set these via the API server configuration file? If so, let's make that clear.

Reply:
Yes, added.